cayenne/src/main/java/org/apache/cayenne/access/ObjectStore.java

```diff
@@ -1011,6 +1013,18 @@ public Collection<ObjectId> getFlattenedIds(ObjectId objectId) {
                 .getOrDefault(objectId, Collections.emptyMap()).values();
     }
 
+    /**
+     * @since 5.0
+     */
+    public Set<Entry<CayennePath,ObjectId>> getFlattenedEntries(ObjectId objectId) {
```

Comment:
I think `Map<CayennePath, ObjectId>` would be a bit cleaner to return here.

Reply:
I thought so too, but didn't know how to name the method then as `getFlattenedMap` didn't seem appropriate.
Any suggestions please ?